### ISSUE TITLE
Pallas FedEx Store Not Selling Ice Boxes

### ISSUE THREAD

**Issue by subbob** [DraconisCluster/DX-Issues#15]
**TL;DR**
-- Informed via ticket that Fed Ex store was only offering Water Tanks for sale.

**Full info**
-- Player sold Ice. Did not see any Ice Boxes for sale. Opened support ticket.
-- DracoNB observed ticket. Guided me in what to look for, found an excess (nearly 11,000 Ice Boxes) in the FedEx Inbound container. None in the Outbound container.
-- Theory: Due to volume changes, and Water Tank listed before Ice Box on whitelist, sorters were filling up with the former and not transferring the latter. (see screenshot)

Draco's explanation to me in **#ticket-discussions**
> It can only move say 100 per refresh due to their size
> It converts a bunch but sorters only move the tanks first so store only offers tanks for sale
> Until it finishes moving all 5000 tanks 100 at a time
> Then it moves the boxes
> Problem seems to be sorter bottleneck for it
> It can only move so many per minute
> And was choosing tanks first, and since there were tanks to move, moved all it could of them

**Corrective Actions**
-- adjusted the 6 industrial conveyor sorters such that 3 only pull Water Tanks and 3 only pull Ice Boxes (see screenshot)
-- did not adjust the lone Conveyor Sorter as I do not know what role it plays
-- created this bug report to capture the change and ensure it's followed up

**Screenshots**

![After Changes](https://github.com/DraconisCluster/DX-Issues/assets/5113030/1eb53845-6e48-4d83-963e-3ab8307af630)

![Inventory Before Changes](https://github.com/DraconisCluster/DX-Issues/assets/5113030/d67d424f-9008-4914-ad78-48ff5062b5b5)

![Inventory After Changes](https://github.com/DraconisCluster/DX-Issues/assets/5113030/b17cc9c5-4604-4b32-b5f7-1733aaf9262e)

**Comment by subbob** [comment]
Another issue with Pallas reported via Ticket. The ticket has not yet been staffed.
![image](https://github.com/DraconisCluster/DX-Issues/assets/5113030/f01b4b3c-6ccd-4dda-bf6e-7085b8e10646)

**Comment by Synirrr** [comment]
Kontu is developing a sorter that should solve this issue

**Comment by draconb** [comment]
This should be resolved for now, we ended up using ISY for now as the super vacuum connector didn't work with the store, it would say the item wasn't available after updating the script to support it.

Would be good to use a more lightweight script / update the store script itself to move stuff around, but selling/buying was very fast when I did it yesterday.

**Comment by draconb** [comment]
Should be resolved with Isy for now
